templates/repo/issue/view_content/pull.tmpl

```diff
@@ -363,8 +363,8 @@
 									{{end}}
 								</div>
 							</div>
-							<div class="dib ml-3">{{$.i18n.Tr "repo.pulls.merge_instruction_hint" | Safe}}</div>
-							<div class="instruct" style="display:none">
+							<div class="instruct dib ml-3">{{$.i18n.Tr "repo.pulls.merge_instruction_hint" | Safe}}</div>
+							<div class="instruct content" style="display:none">
```

Comment:
`content` is too genereic could lead to false CSS matches, I suggest a more unique `instruct-content`.